assertion in check_12 "DeterminantalRepresentations" failing on arm64 and s390x

From builds of the 1.16.99 package for Debian experimental on [arm64](https://buildd.debian.org/status/fetch.php?pkg=macaulay2&arch=arm64&ver=1.16.99%2Bgit40.3a512e5%2Bds-1%7Eexp1&stamp=1609038339&raw=0) and [s390x](https://buildd.debian.org/status/fetch.php?pkg=macaulay2&arch=s390x&ver=1.16.99%2Bgit40.3a512e5%2Bds-1%7Eexp1&stamp=1609027000&raw=0):

```m2
-* running test 12 of package DeterminantalRepresentations in file:
   DeterminantalRepresentations.m2:1527:1:
   rerun with: check_12 "DeterminantalRepresentations" *-

 ulimit -c unlimited; ulimit -t 700; ulimit -m 850000; ulimit -s 8192; ulimit -n 512;  cd /tmp/M2-53666-0/26-rundir/; GC_MAXIMUM_HEAP_SIZE=400M "/<<PKGBUILDDIR>>/M2/usr-dist/s390x-Linux-Debian-buildd-unstable/bin/M2-binary" -q --int --no-randomize --no-readline --silent --stop --print-width 129 -e 'needsPackage("DeterminantalRepresentations", Reload => true, FileName => "/<<PKGBUILDDIR>>/M2/Macaulay2/packages/DeterminantalRepresentations.m2")' <"/tmp/M2-53666-0/25.m2" >>"/tmp/M2-53666-0/25.tmp" 2>&1
/tmp/M2-53666-0/25.tmp:0:1: (output file) error: Macaulay2 exited with status code 1
/tmp/M2-53666-0/25.m2:0:1: (input file)
M2: *** Error 1
-* running test 13 of package DeterminantalRepresentations in file:
   DeterminantalRepresentations.m2:1539:1:
   rerun with: check_13 "DeterminantalRepresentations" *-
../m2/debugging.m2:20:6:(1):[9]: error: test #12 of package DeterminantalRepresentations failed:

o3 = ({.415912}, | -.60855  .442386   -.42001  .489525  .133864   |)
      {.538554}  | -.385718 -.0306554 .39448   -.323399 .768167   |
      {.606588}  | .234815  .751799   .591227  .145593  -.0944111 |
      {.848005}  | -.587034 -.317548  .519553  .143489  -.513838  |
      {.873665}  | -.284854 .370563   -.220209 -.783576 -.345046  |

o3 : Sequence

i4 : assert(clean(eps, A - E#1*diagonalMatrix(E#0)*transpose E#1) == 0)
stdio:6:1:(3): error: assertion failed

../m2/testing.m2:59:22:(1):[8]: --back trace--
../m2/methods.m2:119:80:(1):[7]: --back trace--
../m2/option.m2:16:8:(1):[6]: --back trace--
../m2/testing.m2:44:33:(1):[5]: --back trace--
../m2/methods.m2:119:80:(1):[4]: --back trace--
../m2/option.m2:16:8:(1):[3]: --back trace--
currentString:1:91:(3):[2]: --back trace--
Macaulay2/Core/startup.m2.in:563:33:(0):[1]: --back trace--
Macaulay2/Core/startup.m2.in:674:6:(0): --back trace--
make[4]: *** [Makefile:39: check-DeterminantalRepresentations] Error 1
make[4]: Leaving directory '/<<PKGBUILDDIR>>/M2/Macaulay2/packages'
make[3]: *** [Makefile:14: check-in-packages] Error 2
make[3]: Leaving directory '/<<PKGBUILDDIR>>/M2/Macaulay2'
make[2]: *** [GNUmakefile:229: check-in-Macaulay2] Error 2
make[2]: Leaving directory '/<<PKGBUILDDIR>>/M2'
```

I imagine this can be fixed by more fiddling with epsilons.  I'm surprised that I haven't run into this particular failure before -- this test passes on arm64 and s390x in Ubuntu PPA builds regularly!

I'm running into this same issue, I'm trying to build V1.22 from source on Ubuntu 22.04 x86_64.

Configuring with:
```
./configure --enable-download --prefix=/usr/local/Macaulay2/1.22/  --disable-documentation --enable-build-libraries="csdp mpsolve"
```

During `make check`:

```
/usr/local/src/M2-git/M2/usr-dist/x86_64-Linux-Ubuntu-22.04/bin/M2 -q --no-preload --stop --silent -e "needsPackage(\"DeterminantalRepresentations\",LoadDocumentation=>true,DebuggingMode=>true)" -e "debug Core; argumentMode = defaultMode" -e "check(DeterminantalRepresentations,UserMode=>false,Verbose=>false); exit 0"
 -- capturing check(0, "DeterminantalRepresentations")                -- 0.211848 seconds elapsed
 -- capturing check(1, "DeterminantalRepresentations")                -- 0.218113 seconds elapsed
 -- capturing check(2, "DeterminantalRepresentations")                -- 0.20049 seconds elapsed
 -- capturing check(3, "DeterminantalRepresentations")               -- warning: experimental computation over inexact field begun
--          results not reliable (one warning given per session)
 -- 0.130327 seconds elapsed
 -- capturing check(4, "DeterminantalRepresentations")                -- 0.0983274 seconds elapsed
 -- capturing check(5, "DeterminantalRepresentations")                -- 0.457801 seconds elapsed
 -- capturing check(6, "DeterminantalRepresentations")                -- 0.124389 seconds elapsed
 -- capturing check(7, "DeterminantalRepresentations")                -- 1.37851 seconds elapsed
 -- capturing check(8, "DeterminantalRepresentations")                -- 5.68223 seconds elapsed
 -- capturing check(9, "DeterminantalRepresentations")                -- 5.97214 seconds elapsed
 -- capturing check(10, "DeterminantalRepresentations")               -- 0.680693 seconds elapsed
 -- capturing check(11, "DeterminantalRepresentations")               -- 0.0796518 seconds elapsed
 -- capturing check(12, "DeterminantalRepresentations")               -- capture failed; retrying ...
 -- running   check(12, "DeterminantalRepresentations")              
 ulimit -t 700; ulimit -m 850000; ulimit -s 8192; ulimit -n 512;  cd /tmp/M2-1479133-0/2-rundir/; GC_MAXIMUM_HEAP_SIZE=400M "/usr/local/src/M2-git/M2/usr-dist/x86_64-Linux-Ubuntu-22.04/bin/M2-binary" -q --no-randomize --no-readline --silent --stop --print-width 77 -e 'needsPackage("DeterminantalRepresentations",Reload=>true,FileName=>"/usr/local/src/M2-git/M2/Macaulay2/packages/DeterminantalRepresentations.m2")' <"/tmp/M2-1479133-0/1.m2" >>"/tmp/M2-1479133-0/1.tmp" 2>&1
/tmp/M2-1479133-0/1.tmp:0:1:(3):[11]: (output file) error: Macaulay2 exited with status code 1
/tmp/M2-1479133-0/1.m2:0:1: (input file)
M2: *** Error 1
 -- 0.783174 seconds elapsed
 -- capturing check(13, "DeterminantalRepresentations")               -- 0.107704 seconds elapsed
../m2/debugging.m2:18:6:(1):[9]: error: test(s) #12 of package DeterminantalRepresentations failed.
../m2/testing.m2:126:9:(1):[8]: --back trace--
../m2/methods.m2:154:80:(1):[7]: --back trace--
../m2/option.m2:17:8:(1):[6]: --back trace--
../m2/testing.m2:99:33:(1):[5]: --back trace--
../m2/methods.m2:154:80:(1):[4]: --back trace--
../m2/option.m2:17:8:(1):[3]: --back trace--
currentString:1:1:(3):[2]: --back trace--
../m2/startup.m2.in:562:33:(0):[1]: --back trace--
../m2/startup.m2.in:662:6:(0): --back trace--
make[2]: *** [Makefile:50: check-DeterminantalRepresentations] Error 1
make[2]: Leaving directory '/usr/local/src/M2-git/M2/Macaulay2/packages'
make[1]: *** [Makefile:15: check-in-packages] Error 2
make[1]: Leaving directory '/usr/local/src/M2-git/M2/Macaulay2'
make: *** [GNUmakefile:251: check-in-Macaulay2] Error 2
make: Leaving directory '/usr/local/src/M2-git/M2'
```

```
$ cat /tmp/M2-1479133-0/1.tmp
-- -*- M2-comint -*- hash: 90306928

i1 :  -- /usr/local/src/M2-git/M2/Macaulay2/packages/DeterminantalRepresentations.m2:1527:1: location of test code
      -- cholesky, randomPSD
     eps = 1e-15

o1 = 1e-15

o1 : RR (of precision 53)

i2 : A = randomPSD 5

o2 = | .585298   -.0545901 .0184505  -.0599728  -.149145   |
     | -.0545901 .761001   -.0118983 -.160458   -.02979    |
     | .0184505  -.0118983 .56512    .0605255   -.0250409  |
     | -.0599728 -.160458  .0605255  .609506    -.00367036 |
     | -.149145  -.02979   -.0250409 -.00367036 .761799    |

                5         5
o2 : Matrix RR    <-- RR
              53        53

i3 : E = eigenvectors(A, Hermitian => true)

o3 = ({.415912}, | -.60855  -.442386 .42001   .489525  .133864   |)
      {.538554}  | -.385718 .0306554 -.39448  -.323399 .768167   |
      {.606588}  | .234815  -.751799 -.591227 .145593  -.0944111 |
      {.848005}  | -.587034 .317548  -.519553 .143489  -.513838  |
      {.873665}  | -.284854 -.370563 .220209  -.783576 -.345046  |

o3 : Sequence

i4 : assert(clean(eps, A - E#1*diagonalMatrix(E#0)*transpose E#1) == 0)
stdio:6:1:(3): error: assertion failed

```

--- a/M2/Macaulay2/packages/DeterminantalRepresentations.m2
+++ b/M2/Macaulay2/packages/DeterminantalRepresentations.m2
@@ -1516,7 +1516,7 @@ assert(isOrthogonal(O, Tolerance=>1e-5) and isDoublyStochastic A and clean(1e-8,
 ///
 
 TEST /// -- cholesky, randomPSD
-eps = 1e-15
+eps = 1e-14
 A = randomPSD 5
 E = eigenvectors(A, Hermitian => true)
 assert(clean(eps, A - E#1*diagonalMatrix(E#0)*transpose E#1) == 0)

